repositories/cache/metadata/memcache/memcache_test.go

```diff
@@ -44,6 +44,15 @@ func (s *memcacheTestSuite) TestListContainersError() {
 	s.Require().Equal("some error", err.Error())
 }
 
+func (s *memcacheTestSuite) TestListContainersByPage() {
+	s.repoMock.On("ListContainersByPage", defaultNamespace, uint64(0), uint64(15)).Return(uint64(500), []string{"container1"}, nil).Once()
+
+	total, containers, err := s.cache.ListContainersByPage(s.ctx, defaultNamespace, 0, 15)
```

Comment:
Please add one more call for cache's ListContainers....() since in current test scenario you have mock to ensure it's called once and you calling it once so you're actually not checking if the second call will return you cached data and won't go into the downstream repository.